test/integration/globalization.spec.js

```diff
@@ -0,0 +1,145 @@
+
+      if (!regions.length) return cb();
+
+      // should copied docus to proper dirs
+      regions.forEach((region) => {
+        const createDestinationPath = pathCreator.globalizationDestination(config.skeletonOutDestination, topicDetails, region.code);
+
+        // check copied version dir
+        const destinationPath = createDestinationPath(topicDetails.version, false);
+        stats = fs.statSync(destinationPath);
+        expect(stats.isDirectory()).to.equal(true);
+
+        // check copied latest dir
+        const destinationPathLatest = createDestinationPath('latest', false);
+        if (topicDetails.latest){
+          stats = fs.statSync(destinationPathLatest);
+          expect(stats.isDirectory()).to.equal(true);
+        }
+
+        const isInternal = fs.existsSync(topicDetails.genDocuLocationInternal);
+
+        // check copied to internal
+        const destinationPathInternal = createDestinationPath(topicDetails.version, true);
+        if (isInternal){
+          stats = fs.statSync(destinationPathInternal);
+          expect(stats.isDirectory()).to.equal(true);
+        }
+
+        // check copied latest dir to internal
+        const destinationPathLatestInternal = createDestinationPath('latest', true);
+        if (topicDetails.latest && isInternal){
+          stats = fs.statSync(destinationPathLatestInternal);
+          expect(stats.isDirectory()).to.equal(true);
+        }
+      });
+
+      cb();
+    });
+  });
+
+  it('should replace domain in copied files', (done) => {
+
+    eachRegTopic.async(registry, config, done, (topicDetails, cb) => {
+
+      const regions = _mapMarketsToRegions(topicDetails.markets);
+
+      if (!regions.length) return cb();
+
+      const checkReplaceDomain = [];      
+      regions.forEach((region) => {
+        if(!region.domain)
+          return;
+
+        const createDestinationPath = pathCreator.globalizationDestination(config.skeletonOutDestination, topicDetails, region.code);
+        const destinationPath = createDestinationPath(topicDetails.version, false);
+        checkReplaceDomain.push(_checkReplaceDomain(destinationPath));
+      });
+
+      async.series(checkReplaceDomain, cb);
+    });
+  });
+
+  after((done) => {
+    rimraf(`${config.tempLocation}`, done);
+  });
+
+});
+
+function _checkReplaceDomain(destinationPath){
+  return function(cb){
+    gulp.src(`${destinationPath}/**/*`)
+      .pipe(tap((file) => {
+        if(!file.contents)
+          return;
+        const contentFile = String(file.contents);
+        const isReplacedDomain = contentFile.indexOf(config.defaultBaseUriDomain) === -1;
+        expect(isReplacedDomain).to.equal(true);
+      }))
+      .on('end', cb);
+  };
+}
+
+function _mapMarketsToRegions(markets){
+  if(!markets.length) return false;
+  return markets.map((market) => {
+    return {
+      code: market,
+      domain: market === 'us' ? `${market}.domain` : null
```

Comment:
it is adjust to tests registry, globalization function should replace string domain in files only if  exists domain to replace in region object